src/modules/shared/Header/Header.tsx

```diff
@@ -92,10 +90,25 @@ export const Header: React.FC = () => {
       </div>
 
       <div className={styles.header__right}>
+        <div className={styles.header__themeSwitch}>
+          <input
+            type="checkbox"
+            className={styles.header__themeSwitch_input}
+            id="themeSwitch"
+            onChange={() => ThemeHandler()}
+          />
```

Comment:
its ok, but you can reduce onChange={ThemeHandler} and why with a capital letter? it`s not a function-constructor